### PR TITLE
WIP - OWLS-73066 - - WLDF SCALING IS NOT WORKING ON OPERATOR JENKINS RUN

### PR DESCRIPTION
**Pull request by alai8**
Get WLDF scaling test to pass in jenkins by updating domain crd version in k8s rest api call in scalingAction.sh

Passed jenkins http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/1382/

### CODE REVIEW

File: src/scripts/scaling/scalingAction.sh [hunk 1]
```diff
@@ -118,7 +118,7 @@ port=`echo ${STATUS} | python cmds.py`
 echo "port: $port" >> scalingAction.log
 
 # Reteive Custom Resource Domain 
-DOMAIN=`curl -v --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" $kubernetes_master/apis/weblogic.oracle/v2/namespaces/$wls_domain_namespace/domains/$domain_uid`
+DOMAIN=`curl -v --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" $kubernetes_master/apis/weblogic.oracle/v3alpha1/namespaces/$wls_domain_namespace/domains/$domain_uid`
```
Comment (lennyphan) [inline src/scripts/scaling/scalingAction.sh]: wonder if we should parameterize the version so we don't have to keep updating the script?  can we retrieve the version?

Comment (rjeberhard) [reply src/scripts/scaling/scalingAction.sh]: We'll need to parameterize and pass in this value as the version will change depending on the domain.  For instance, if the customer had a domain resource with version "v2" and upgraded to 2.1, then the domain version would still be "v2".  The 2.1 operator will support both "v2" and "v3" (currently, "v3alpha1").

Comment (rjeberhard) [reply src/scripts/scaling/scalingAction.sh]: The value we need here is metadata.selfLink from the domain resource

File: src/scripts/scaling/scalingAction.sh [hunk 2]
```diff
@@ -118,7 +118,7 @@ port=`echo ${STATUS} | python cmds.py`
 echo "port: $port" >> scalingAction.log
 
 # Reteive Custom Resource Domain 
-DOMAIN=`curl -v --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" $kubernetes_master/apis/weblogic.oracle/v2/namespaces/$wls_domain_namespace/domains/$domain_uid`
```
Comment (markxnelson) [inline src/scripts/scaling/scalingAction.sh]: you also need to update the copyright notice please

Comment (alai8) [reply src/scripts/scaling/scalingAction.sh]: Thanks. Missed that. Updated now.

File: src/scripts/scaling/scalingAction.sh (hunk 1 above)
Comment (alai8) [reply src/scripts/scaling/scalingAction.sh]: Updated script to query the domain CRD version to be used in the "retrieve custom resource domain" API call.

Comment (rjeberhard) [reply src/scripts/scaling/scalingAction.sh]: Based on my testing, the implementation you picked, @alai8, is the correct one.  Nice work!

**Pull request merged**